keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -484,27 +501,27 @@ the existing defaulting functions
 2. Update kube-openapi to parse the `+default` marker and include it in
 the OpenAPI definition.
 3. Implement the new CRD semantics that clears `null` values from
-objects when the `x-kubernetes-remove-nulls` flag is present and true at
-the top-level (or in the CRD).
+objects when nullable is unset or false.
```

Comment:
Implement the new CRD semantics for explicit `null` values when `nullable` is unset or false:
* properties and list items with a default have the default applied, replacing the explicit null value
* list items without a default let the explicit null value remain, resulting in a validation error (matches current behavior)
* properties without a default have the explicit `null` value removed